static/src/javascripts/projects/common/views/svg.js

```diff
@@ -0,0 +1,24 @@
+define(function () {
+    return svg;
+
+    function svg(markup, classes, title) {
+        // Only mess with classes if we actually need to.
+        if (classes) {
+            if (Array.isArray(classes)) {
```

Comment:
I've probably asked this many times. Do we include a polyfill for `Array.isArray` in IE8? can't remember